### DISCUSSION TITLE
Build fails with Module parse failed: Unexpected character ' ' (1:0) Error

### DISCUSSION THREAD

**Discussion by Simonaester** [facebook/docusaurus#9022] [Q&A] [answered]
![buildError](https://github.com/facebook/docusaurus/assets/61542974/f7d172da-1c18-4d29-a9af-d714d124522f)

As you can see, the error log does not specify where the problem may have occurred. Client side compiles without errors, but the build keeps breaking and I really don't know how to fix it.
Since in a section of my site I import a package with a custom theme (which includes a custom font), I tried to install the ['docusaurus-plugin-fontloader'](https://www.npmjs.com/package/docusaurus-plugin-fontloader) plugin, but the problem still persists.

**My environment:**

    Public source code: https://github.com/selfcommunity/developers-doc
    Public site URL: https://developers.selfcommunity.com/
    Docusaurus version used: 2.4.1
    Environment name and version (e.g. Chrome 89, Node.js 16.4):
        node --version: v18.15.0

**Steps to reproduce:**

1. Clone [project](https://github.com/selfcommunity/developers-doc.git) with its submodules 
    (git clone --recurse-submodules https://github.com/selfcommunity developers-doc.git)
2. Move to directory  sdk/community-js and run yarn bootstrap
3. Run yarn install (project root)

**Comment by slorber** [comment]
Hi

Your project is complex, relies on external dependencies not under our control.

Also it's apparently been a few days since your project CI is failing, initially not with the same error message. And even when it was successful the CI reported many warnings.

I can't help you with such repro unfortunately. The problem might be due to your internal `@selfcommunity` packages (or site code) requiring/importing some files with extensions that Docusaurus does not support by default, and you need to configure extra webpack loaders to handle those extensions. 

You can try to remove things from your site one by one until things start working again to figure out what might be the problem.

**Reply by slorber** [reply, marked as answer]
@Simonaester that's unfortunate that Docusaurus is unable to tell you which file has the bad import, and this part is definitively a problem on our side.

I suggested workarounds to find the problematic file here:
https://github.com/facebook/docusaurus/issues/9023#issuecomment-1568574913

**Reply by Simonaester** [reply]
Thank you very much, I finally managed to trace the file that caused the error.
